docs/_docs/Unofficial Sidecar Academy/Sidecar-Academy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: LDSA Unofficial Sidecar Academy
+category: Unofficial Sidecar Academy
+order: 1
+---
+
+Contents:
+- [Course Overview](#course-overview)
+- [What will be covered](#what-will-be-covered)
+- [What is the same and what is different](#what-is-the-same-and-what-isdifferent)
+
+---
+
+# Course overview
+Over the last few years, many people have wanted to but not been able to get into the LDSSA. As of now, we have not been able to offer them any community-based alternatives which leaves them with the usual options of lonely online courses. While the course material, portal, and access to instructors are definitely useful things, the vast majority of the value provided to the students is in the community aspect. The feeling that you're not going it alone is incredibly powerful.
+
+Since 2020, we have started to offer the LDSA Unofficial Sidecar Academy, a community-based lightweight version of the LDSA and is meant to provide all of those who didn't have a chance to get into the 2020 LDSSA, a concrete next step in their pursuit of Data Science.
+
```

Comment:
I would add "It should not be mistaken for the Lisbon Data Science Academy, or the Starters Course."